docs/changelog/migration_2023_01_0.rst

```diff
@@ -0,0 +1,30 @@
+=======================================
+What's new in buildstockbatch 2023.01.0
+=======================================
+
+.. admonition:: About this Document
+
+    This document describes changes between buildstockbatch version 2022.10.1 and
+    buildstockbatch version 2023.01.0
+
+General
+=======
+
+This version should be backwards compatible with previous versions of
+buildstockbatch. The default version of OpenStudio was changed to 3.5.1, so if
+you require an older version, make sure it is called out in your project file.
+
+See :doc:`changelog_2023_01_0` for details of changes since the last version.
+
+Residential HPXML Workflow Changes
+==================================
+
+Two new optional arguments were added to the Residential HPXML Workflow
+Generator. It is recommended to add the ``timeseries_num_decimal_places``
```

Comment:
@nmerket Are you basing this recommendation on @rajeee's comment [here](https://github.com/NREL/resstock/issues/1016#issuecomment-1376693846)? Perhaps we want to change this suggestion to 3 based on his more recent comment [here](https://github.com/NREL/resstock/issues/1016#issuecomment-1404212345)?

Reply:
Yeah, that's what I was thinking. I'm going to sidestep this by referring people to the workflow generator documentation instead.